src/modules/tokens/hooks/useBalancesAndAllowances.ts

```diff
@@ -12,7 +12,7 @@ export function useBalancesAndAllowances(params: BalancesAndAllowancesParams): B
   const balancesAndAllowances = useOnchainBalancesAndAllowances(params)
 
   // TODO: This function still has too many re-renders, we shold investigate (for now, focusing on only the re-factor)
-  console.debug('[usebalancesAndAllowances] Get balancesAndAllowances', params, balancesAndAllowances)
+  // console.debug('[usebalancesAndAllowances] Get balancesAndAllowances', params, balancesAndAllowances)
```

Comment:
![](https://1.bp.blogspot.com/-pIAr3zVoTag/Xeo_XZM1EiI/AAAAAAAAJjA/7aMBSGfL128SbXrrwURToINC3rRRb1G4gCLcBGAsYHQ/s1600/Thank-You-GIF%2B%252833%2529.gif)

Reply:
Thanks from me too 🙌